### PR TITLE
Make node start options configurable

### PR DESCRIPTION
**Pull request by celebdor**
Thanks to this, the node startup can be customized.

Change-Id: Ifb93bf892d7de52a575b8d0231328d44f733d0aa
Signed-off-by: Antoni Segura Puimedon <celebdor@gmail.com>

### CODE REVIEW

**Comment by sdodson** [comment]
bot, add author to whitelist

**Comment by sdodson** [comment]
aos-ci-test

File: roles/openshift_node/tasks/config/configure-node-settings.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     create: true
   with_items:
   - regex: '^OPTIONS='
-    line: "OPTIONS=--loglevel={{ openshift.node.debug_level | default(2) }}"
+    line: "OPTIONS=--loglevel={{ openshift.node.debug_level | default(2) }} {{ openshift.node.start_options }}"
```
Comment (sdodson) [inline roles/openshift_node/tasks/config/configure-node-settings.yml]: We're trying to move away from openshift_facts so lets forego this. So instead just `{{ openshift_node_start_options | default('') }}`

File: roles/openshift_node/tasks/main.yml [hunk 2]
```diff
@@ -23,6 +23,7 @@
       local_facts:
         annotations: "{{ openshift_node_annotations | default(none) }}"
         debug_level: "{{ openshift_node_debug_level | default(openshift.common.debug_level) }}"
+        start_options: "{{ openshift_node_start_options | default(None) }}"
```
Comment (sdodson) [inline roles/openshift_node/tasks/main.yml]: And remove this.

File: roles/openshift_node/tasks/config/configure-node-settings.yml (hunk 1 above)
Comment (celebdor) [reply roles/openshift_node/tasks/config/configure-node-settings.yml]: Sounds good. I'll update! And I'm glad we're moving away from the extra level of indirection.

**Comment by sdodson** [comment]
aos-ci-test

**Pull request merged**